utils/lpac/Makefile

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: AGPL-3.0-only
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lpac
+PKG_VERSION:=2.0.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=lpac-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/estkme-group/lpac/tar.gz/refs/tags/v$(PKG_VERSION)?
+PKG_HASH:=686faad1cac49649aaa749be41b44f6ec303833d4f8ef586245de4ad545d4886
+
+PKG_MAINTAINER:=David Bauer <mail@david-bauer.net>
+PKG_LICENSE:=AGPL-3.0-only
```

Comment:
Isn't the package dual licensed according to https://github.com/estkme-group/lpac/blob/f521c455dc18459a719e4e90548829a8c9b10dfc/CMakeLists.txt#L40